source/includes/_account.md

```diff
@@ -173,12 +173,14 @@ The body of the request is a JSON string specifying any attributes of the resour
 | **timeZone** | "UTC" |
 | all others | null |
 
-All attributes are optional, so it is possible to quickly create a shell account. This account could be used for test purposes, or so its attributes can be filled in later. This also ensures the initial account contains no PII. Note, however, that the `currency` attribute must have a non-null value, which can be added later, for invoicing to work properly.
+All attributes (other than name) are optional, so it is possible to quickly create a shell account. This account could be used for test purposes, so its attributes can be filled in later. This also ensures the initial account contains no PII. Note, however, that the `currency` attribute must have a non-null value. This cannot be added later and is required for invoicing to work properly.
```

Comment:
Name should be optional. Was it required during your testing?

Reply:
Earlier, I got an error when I ran it without specifying a name, however, I'm not able to reproduce it now, it works fine even when a name is not specified. So, I've reverted my change. Sorry about this one.